NuGet.Docs/ndocs/API/NuGet.exe-Credential-Providers.md

```diff
@@ -0,0 +1,138 @@
+# NuGet.exe Credential Providers
+
+NuGet.exe 3.3+ now supports Credential Providers, which enable NuGet to work seamlessly with authenticated feeds. 
+After you install a NuGet.exe Credential Provider, NuGet.exe will automatically acquire and refresh credentials for authenticated feeds as necessary.
+
+Note: NuGet.exe Credential providers only work in NuGet.exe (not in dotnet restore or Visual Studio). To learn more about NuGet Credential Providers for Visual Studio, check out [this page](Visual-Studio-Credential-Providers.md).
```

Comment:
Addressed in f24d41dd85e5a571391eccbf98010547da983a03. Also updated the already existing page to use the info call out.